packages/scrollable-image/src/constants/index.js

```diff
@@ -0,0 +1,4 @@
+export const packageName = 'scrollable-image'
+export const namespace = '__twreporterEmbeddedData'
+export const cdnLinkPrefix =
+  'https://unpkg.com/@twreporter/orangutan@latest/dist/'
```

Comment:
We could use sub-pacakage level cdn link prefix
`https://unpkg.com/@twreporter/scrollable-image@latest/dist/`